.github/actions/check-status/__tests__/poll.test.ts

```diff
@@ -0,0 +1,138 @@
+import { poll } from "../src/poll";
+
+const octokit = {
+  request: jest.fn(),
+};
+
+const run = () =>
+  poll({
+    octokit: octokit as any,
+    log: () => {},
+    statusURL: "repos/owner/repo-name/commits/ref/status",
+    owner: "owner",
+    repo: "repo-name",
+    ref: "ref",
+    timeoutSeconds: 3,
+    intervalSeconds: 1,
+  });
+
+describe("poll", () => {
```

Comment:
neat, when do these actually run? I didn't see them in the vercel logs. Once our tests get more extensive i'd be down to create a separate testrunner (separate from builds, to keep those fast)